python/dgl/distributed/partition.py

```diff
@@ -24,6 +34,43 @@
     ETYPE: F.int32
     }
 
+def _format_part_metadata(part_metadata, formator):
+    ''' Format etypes with specified formator
+    '''
```

Comment:
No whitespace after `'''`. New newline after a short description. Miss `.` at the end.